rcl/Doxyfile

```diff
@@ -0,0 +1,17 @@
+# All settings not listed here will use the Doxygen default values.
+
+PROJECT_NAME           = "rcl"
+PROJECT_NUMBER         = beta1
```

Comment:
Ok, I'll put in `master` in the pr. If I had more time I would make it into a CMake custom command and template the file with the commit hash or branch or tag automatically and handle the tag files through the build/install spaces.